tests/test_api_urls.py

```diff
@@ -11,16 +11,22 @@
 import sys
 import unittest
 
+from mock import patch, MagicMock
+from requests import Response
+from werkzeug import wsgi
+
 # Add webcompat module to import path
 sys.path.append(os.path.realpath(os.pardir))
 import webcompat
```

Comment:
It's not part of your code but a good opportunity to clean it up. 
We are not forced to do that. We can just use 

```python
  import webcompat 
```
see

```diff
→ git diff
diff --git a/tests/test_api_urls.py b/tests/test_api_urls.py
index 2751c22..9c90ccd 100644
--- a/tests/test_api_urls.py
+++ b/tests/test_api_urls.py
@@ -8,15 +8,11 @@
 
 import json
 import os.path
-import sys
 import unittest
 
 from mock import patch, MagicMock
 from requests import Response
-from werkzeug import wsgi
 
-# Add webcompat module to import path
-sys.path.append(os.path.realpath(os.pardir))
 import webcompat
 
 FIXTURES_PATH = os.getcwd() + '/tests/fixtures'
```

then

```
→ nosetests tests/test_api_urls.py
.........
----------------------------------------------------------------------
Ran 9 tests in 0.017s

OK
```